src/components/tag/tag.classnames.ts

```diff
@@ -1,80 +1,365 @@
-import { TagType, TagSize, TagVariant } from '.'
+import { cva } from 'class-variance-authority'
 import {
   LabelPairedTriangleExclamationBoldIcon,
   LabelPairedCircleExclamationBoldIcon,
   LabelPairedCircleCheckBoldIcon,
   LabelPairedCircleInfoBoldIcon,
+  LabelPairedTriangleExclamationRegularIcon,
+  LabelPairedCircleExclamationRegularIcon,
+  LabelPairedCircleCheckRegularIcon,
+  LabelPairedCircleInfoRegularIcon,
   QuillSvgProps,
 } from '@deriv/quill-icons'
+import { TPreset, TSize } from '.'
 
-type TypeTagColor = {
-  [key in TagType]: {
-    [key in TagVariant]: string
+type TTagIcon = {
+  [key in Exclude<TPreset, 'custom'>]: {
+    [key in 'true' | 'false']: React.ForwardRefExoticComponent<
+      Omit<QuillSvgProps, 'ref'>
+    >
   }
 }
 
-export const TagColors: TypeTagColor = {
+export const TagIcons: TTagIcon = {
   error: {
-    fill: 'bg-solid-red-100 text-solid-red-800',
-    outline: 'border-75 border-solid-red-700 text-solid-red-800',
+    true: LabelPairedTriangleExclamationBoldIcon,
+    false: LabelPairedTriangleExclamationRegularIcon,
   },
   warning: {
-    fill: 'bg-solid-orange-100 text-solid-orange-800',
-    outline: 'border-75 border-solid-orange-700 text-solid-orange-800',
+    true: LabelPairedCircleExclamationBoldIcon,
+    false: LabelPairedCircleExclamationRegularIcon,
+  },
+  success: {
+    true: LabelPairedCircleCheckBoldIcon,
+    false: LabelPairedCircleCheckRegularIcon,
   },
   info: {
-    fill: 'bg-solid-blue-100 text-solid-blue-800',
-    outline: 'border-75 border-solid-blue-700 text-solid-blue-800',
+    true: LabelPairedCircleInfoBoldIcon,
+    false: LabelPairedCircleInfoRegularIcon,
```

Comment:
Maybe there is a reason behind choosing `true`/`false` here but if it's about selecting the icons then wouldn't it be better to keep the keys as `bold`/`regular`? Right now there are just these 2 but in the future, there could be some other variations?